### PR TITLE
52389-Fix_signup

### PR DESCRIPTION
**Pull request by htpt2910**
## Related Tickets
- [#52389](https://edu-redmine.sun-asterisk.vn/issues/52389)

## WHAT (optional)
- Fix Sign up 

## HOW
- Add sign up component + sign up layout

### CODE REVIEW

File: spec/components/form_component_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```
Comment (l0ngvh) [inline spec/components/form_component_spec.rb]: Xóa mấy file trống này luôn

File: spec/components/signup_component_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```
Comment (l0ngvh) [inline spec/components/signup_component_spec.rb]: Xóa đây nữa.

**Review by l0ngvh — COMMENTED** [review]
comment

**Comment by htpt2910** [comment]
ready

**Pull request merged**